src/views/EmailVerif.vue

```diff
@@ -0,0 +1,48 @@
+<script setup lang="ts">
+import axios from "axios"
```

Comment:
Try not to keep unused import in your code as it slows down page loading (and linters don't like this)